src/test/java/com/google/api/codegen/testdata/discoveries/py/py_appengine.v1beta4.json.baseline

```diff
@@ -0,0 +1,472 @@
+
+"""
+PRE-REQUISITES:
+---------------
+1. If not already done, enable the Google App Engine Admin API and check quota for your project at
+   https://console.developers.google.com/apis/api/appengine_component/quotas
+2. Install python client library for Google APIs using 'pip install --upgrade google-api-python-client'
+3. This sample uses Application Default Credentials for Auth. If not already done, install the gcloud CLI from
+   https://cloud.google.com/sdk/ and run 'gcloud beta auth application-default login'
+"""
+
+from __future__ import print_function
+import json
+
+from apiclient import discovery
+from oauth2client.client import GoogleCredentials
+
+# Authentication is provided by gcloud tool when running locally
+# and by built-in service accounts when running on GAE, GCE or GKE.
+# See https://developers.google.com/identity/protocols/application-default-credentials for more information.
+credentials = GoogleCredentials.get_application_default()
+
+# Construct the appengine service object (version v1beta4) for interacting
+# with the API. You can browse other available API services and versions at
+# https://developers.google.com/api-client-library/python/apis/
+service = discovery.build('appengine', 'v1beta4', credentials=credentials)
+
+
+# TODO: Change placeholders below to values for parameters to the 'get' method:
+# * Part of `name`. Name of the application to get. For example: "apps/myapp".
```

Comment:
You use back ticks here but regular ticks below in the TODO. I suggest regular quotes everywhere if that's possible.